com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -15,6 +15,36 @@ public enum MixedLightingSetup
         Subtractive,
     };
 
+    /// <summary>
+    /// Enumeration that indicates what kind of image scaling is occurring if any
+    /// </summary>
+    public enum ImageScaling
+    {
```

Comment:
@sandy-carter-unity how does this fit into the dynamic scaling work in progress?

Reply:
The dynamic scaling work is downscaling only.
It uses viewports of already allocated targets.